sos/plugins/openswan.py

```diff
@@ -42,6 +42,9 @@ def setup(self):
         if self.get_option("ipsec-barf"):
             self.add_cmd_output("ipsec barf")
 
-        self.add_forbidden_path("/etc/ipsec.d/key[3-4].db")
+        self.add_forbidden_path('/etc/ipsec.secrets',
```

Comment:
Missing the `[]` in this call...:

```
Traceback (most recent call last):
  File "/home/jake/projects/sos/sos/sosreport.py", line 1033, in setup
    plug.setup()
  File "/home/jake/projects/sos/sos/plugins/openswan.py", line 42, in setup
    '/etc/ipsec.d/*.secrets')
TypeError: add_forbidden_path() takes exactly 2 arguments (5 given)
```

Reply:
Aw jeez.. I have new glasses this week. I swear they have made me blind - since you need to respin to list-ise the members anyway, it would be better to put this in the `sos` list call style:

```python
        self.add_forbidden_path([
            "first_path",
            "second_path",
            "last_path"
        ])
```

Reply:
> Missing the `[]` in this call...:
Sorry, tests didn't catch this.  Updated.

> Aw jeez.. I have new glasses this week. I swear they have made me blind - since you need to respin to list-ise the members anyway, it would be better to put this in the `sos` list call style:
Should be updated appropriately, let me know if I missed anything.  Thanks!

Reply:
Thanks! And sorry for the fuzzy reviews this week :-)